core/services/relay/evm/tron_transmitter.go

```diff
@@ -30,16 +33,25 @@ func (t *tronEVMTransmitterWrapper) LatestTransmissionDetails(ctx context.Contex
 	return configDigest, epoch, 0, nil, time.Time{}, nil
 }
 
-func NewTronContractTransmitter(ctx context.Context, lggr logger.Logger, evmTransmitter ContractTransmitter, ethKeystore keys.Store, configWatcher *configWatcher, opts configTransmitterOpts, transmissionContractABI abi.ABI, ocrTransmitterOpts ...OCRTransmitterOption) (ContractTransmitter, error) {
+// TronContractTransmitterOpts contains the configuration options for creating a Tron contract transmitter
+type TronContractTransmitterOpts struct {
+	EVMTransmitter        ContractTransmitter
+	Keystore              keys.Store
+	ConfigWatcher         *configWatcher
+	ConfigTransmitterOpts configTransmitterOpts
+}
+
+// NewTronContractTransmitter creates a new ContractTransmitter for Tron chains
+func NewTronContractTransmitter(ctx context.Context, lggr logger.Logger, opts TronContractTransmitterOpts, ocrTransmitterOpts ...OCRTransmitterOption) (ContractTransmitter, error) {
```

Comment:
Everything except the context should be moved to `TronContractTransmitterOpts` (logger and  the OCRTransmitterOption array) - that's the point of the structure, to group and document these input arguments.